docs/_index.mdx

```diff
@@ -61,7 +61,7 @@ Nitric is unique in its ability to allow development of applications, which use
 
 Instead of integrating directly with cloud provider APIs for services such as compute, events, queues, documents, storage and caching, Nitric provides a single set of APIs for every supported service. These APIs are fulfilled by a lightweight translation layer, connecting to those services on your behalf. Translation is implemented via plugins, enabling complete flexibility without changing application code.
 
-> Learn about the cloud service [APIs offered by Nitric](intro/concepts/nitric-apis) and how they run in each cloud.
+> Learn about the cloud service [APIs offered by Nitric](/docs/intro/concepts/nitric-apis) and how they run in each cloud.
```

Comment:
cloud services? or each cloud service

Reply:
This is referring to our unified APIs.  Would you prefer:  `Learn about the unified cloud service [APIs..`